site/templates/collection/carousel/item/image.jet

```diff
@@ -1,5 +1,14 @@
 {{block carouselItemImage()}}
   {{if isset(.Images["Carousel"])}}
-    <s72-image src="{{.Images.Carousel}}" alt="{{.Title}}" class="carousel-item-image"></s72-image>
+    {* If carousel focus is set for film (film infos metadata) or tv season (tv seasons metadata) or custom fields for Pages / Bundles then it will set a class accordingly to position the carousel image *}
+      {{carouselFocus = ""}}
+      {{if isset(.InnerItem["CarouselFocus"]) && .InnerItem["CarouselFocus"] != "" }}
+        {{carouselFocus = .InnerItem["CarouselFocus"]}}
+      {{end}}
+      {{ if .InnerItem.CustomFields.GetString("carousel_focus", "") != ""}}
```

Comment:
Why are you looking in CustomFields?  The kibble models have CarouselFocus as a top level prop.

Reply:
It only has it in the models for tvseason / film. pages and bundles its using custom fields since it ended up that they didnt have a metadata field in the database, only custom fields

Reply:
Ah right... well it won't matter anyway if you do what I've suggested in the other comment.